Cargo.toml

```diff
@@ -1,8 +1,13 @@
 [package]
 name = "iota-wallet"
 version = "0.2.0"
-authors = ["Lucas Nogueira <lucas.nogueira@iota.org>"]
-edition = "2018"
+authors = ["IOTA Stiftung"]
```

Comment:
For the author email should we put `contact@iota.org`?

Reply:
The change is borrowed from [bee](https://github.com/iotaledger/bee/blob/dev/bee-common/bee-common/Cargo.toml#L4); perhaps let them know this concern too.